packages/dumb/src/components/stats/simple/simple-stat.utils.ts

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (c) 2022, Diabeloop
+ *
+ * All rights reserved.
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions are met:
+ *
+ * 1. Redistributions of source code must retain the above copyright notice, this
+ *    list of conditions and the following disclaimer.
+ *
+ * 2. Redistributions in binary form must reproduce the above copyright notice,
+ *    this list of conditions and the following disclaimer in the documentation
+ *    and/or other materials provided with the distribution.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+ * AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+ * IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+ * DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE
+ * FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+ * DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+ * SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+ * CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
+ * OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+export const getPercentagePrecision = (percentage: number): number => {
+  // We want to show extra precision on very small percentages so that we avoid showing 0%
+  if (percentage > 0 && percentage < 0.5) {
+    return percentage < 0.05 ? 2 : 1
```

Comment:
(can be optional) I still find it weird to have that many hardcoded values in a file, what do you think of this change?

```ts
const MINIMUM_PERCENTAGE_VALUE = 0
const MAXIMUM_PERCENTAGE_VALUE = 0.5
const PERCENTAGE_PRECISION_BREAKPOINT = 0.05
const BIG_PRECISION = 2
const SMALL_PRECISION = 1
const DEFAULT_PRECISION = 0

export const getPercentagePrecision = (percentage: number): number => {
  if (percentage > MINIMUM_PERCENTAGE_VALUE && percentage < MAXIMUM_PERCENTAGE_VALUE) {
    return percentage < PERCENTAGE_PRECISION_BREAKPOINT ? BIG_PRECISION : SMALL_PRECISION
  }
  return DEFAULT_PRECISION
} 
```